java/src/jmri/jmrix/loconet/Lnsv1DevicesManager.java

```diff
@@ -44,12 +44,15 @@
  */
 
 public class Lnsv1DevicesManager extends PropertyChangeSupport
-        implements LocoNetListener {
+        implements LocoNetListener, jmri.Disposable {
     private final LocoNetSystemConnectionMemo memo;
     @GuardedBy("this")
     private final Lnsv1Devices lnsv1Devices;
 
-    public Lnsv1DevicesManager(LocoNetSystemConnectionMemo memo) {
+    // constant for thread name, with memo prefix appended.
+    static final String ROSTER_THREAD_NAME = "rosterMatchingListLnsv1DM";
+
+    public Lnsv1DevicesManager(@javax.annotation.Nonnull LocoNetSystemConnectionMemo memo) {
```

Comment:
I personally prefer in this case Nonnull, instead of the complete package, since that makes the code easier to read.